app/parsers/bulkrax/csv_parser.rb

```diff
@@ -209,11 +209,31 @@ def key_allowed(key)
 
     # All possible column names
     def export_headers
+      # need to find the numerated headers, which may be different per entry
+      # and then remove the duplicates
+      parsed_metadata_keys = []
+      importerexporter.entries.where(identifier: current_work_ids)[0..limit || total].each do |entry|
```

Comment:
couldn't think of another way to do this. unsure if this will be super slow when exporting 1000 works or something though.